### PR TITLE
Added FN F2000 rifle

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

SUMMARY: Content "Add FN F2000 assault rifle"

### CODE REVIEW

**Comment by ymber** [comment]
F2000s don't belong in the setting. They're restricted title 2 weapons and they've never been in service with the American military or any police forces in the region.

**Comment by LZhangy** [comment]
> F2000s don't belong in the setting. They're restricted title 2 weapons and they've never been in service with the American military or any police forces in the region.

So by that logic, power armor and laser rifles also shouldn't exist because they aren't used by the American military? Not to mention there is a whole branch of FN in the United States and we already have another foreign rifle like the FAMAS in the game? Your reasoning as to why the lack of usage of this weapon by the majority of the armed forces in the United States equates to it being unsuitable as content isn't very convincing.

**Comment by FuelType-Memes** [comment]
I would point out, that "if we have X we can't we add Y?" line of reasoning in general leads to the removal or correction of X

**Comment by Night-Pryanik** [comment]
There is already #36838 adding almost the same weapon.

File: data/json/items/gun/223.json [hunk 1]
```diff
@@ -144,7 +155,47 @@
...
+  },
+  {
+    "id": "fn_f2000",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "FN F2000" },
+    "description": "A Belgian bullpup rifle chambered in NATO 5.56x45mm rounds, this assault rifle features a compact and modular design resembling a tuna fish.",
+    "weight": "3400 g",
+    "volume": "1700 ml",
```
Comment (LaVeyanFiend) [inline data/json/items/gun/223.json:173]:
```suggestion
    "weight": "3319 g",
    "volume": "7986 ml",
```

File: data/json/items/gun/223.json [hunk 2]
```diff
@@ -144,7 +155,47 @@
...
+    "modes": [
+      [ "DEFAULT", "semi-auto", 1 ],
+      [ "AUTO", "auto", 4 ]
+    ],
+    "built_in_mods": [ "barrel_ported", "grip_mount", "rail_mount", "sights_mount", "tele_sight" ],
+    "magazines": [
+      [
+        "223",
+        [ "stanag30", "stanag50", "survivor223mag" ]
+      ]
```
Comment (LaVeyanFiend) [inline data/json/items/gun/223.json:197]:
```suggestion
    "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ],
    "magazine_well": "228 ml"
```

**Comment by ymber** [comment]
Not an appropriate content addition. Closing in favor of #36838.

**Pull request closed without merging**